vite.config.mts

```diff
@@ -30,23 +31,28 @@ interface Config {
 export default defineConfig(({ mode }) => {
   const env = loadEnv(mode, process.cwd(), '')
   const configDir = `./configs/${env.VITE_SITE_ID}`
+  const configFiles = []
 
   // Load base config
-  const configFileUrl = new URL(`${configDir}/config.yaml`, import.meta.url)
-  let config = load(readFileSync(configFileUrl, 'utf-8')) as Config
+  const mainConfigFile = `${configDir}/config.yaml`
+  configFiles.push(mainConfigFile)
+  let config = load(
+    readFileSync(new URL(mainConfigFile, import.meta.url), 'utf-8')
```

Comment:
Do we need URL here? Why not simply `mainConfigFile`?

Reply:
It's used to reliably resolve the file path to an absolute path, across platforms and in a ES module context. It might not be strictly required but I'd keep it for safety sake.